packages/checkout/sdk/src/widgets/definitions/parameters/addFunds.ts

```diff
@@ -0,0 +1,21 @@
+import { WidgetLanguage } from '../configurations';
+
+export type AddFundsWidgetParams = {
+  /** The language to use for the Add Funds widget */
+  language?: WidgetLanguage;
+
+  /** Configure to show on-ramp option */
+  showOnrampOption?: boolean;
+
+  /** Configure to show swap option */
+  showSwapOption?: boolean;
+
+  /** Configure to show on bridge option */
+  showBridgeOption?: boolean;
+
+  /** Token address of the fund to be added */
+  tokenAddress?: string;
```

Comment:
I'll suggest to improve the naming by prefixing with `from` and `to` since the widget will support LiFi
- toToken
- toAmount